src/opendap/auth/UrsIdPTest.java

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
I copied this heading verbatim from another test file; lmk if it should be altered at all.